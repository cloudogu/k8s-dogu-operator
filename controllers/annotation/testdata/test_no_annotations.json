{
  "ExposedPorts": {
    "80/tcp": {}
  }
}