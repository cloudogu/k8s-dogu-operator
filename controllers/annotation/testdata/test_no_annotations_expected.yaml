# This yaml is exactly as the default input yaml as no annotation were generated. Only the annotations are compared.
apiVersion: v1
kind: Service
metadata:
  name: serviceName
spec:
  selector:
    name: serviceName
  ports:
    - protocol: TCP
      port: 80